src/app/containers/MediaPlayer/index.jsx

```diff
@@ -76,7 +76,7 @@ const MediaPlayerContainer = ({
     return null; // this should be the holding image with an error overlay
   }
 
-  const placeholderSrc = getPlaceholderSrc(imageUrl);
+  const placeholderSrc = getIChefURL(imageBlock);
```

Comment:
It'd be safer to explicitly pull out the `originCode, locator, resolution` values from the imageBlock and explicitly pass them to the `getIChefURL` function. This way, if the structure of the imageBlock is changed, we can easily update it.

Reply:
Makes sense.